.env

```diff
@@ -1,4 +1,4 @@
-APP_BACKEND_URL=https://st-storefront.dev.govirto.com
+APP_BACKEND_URL=http://localhost:2082
```

Comment:
Please, remove this file from the request. It's better to use `.env.local` file to override your local settings keeping them gitignored.

Reply:
revert it 👍